src/Uno.UI/Controls/NavigationBarHelper.Android.cs

```diff
@@ -0,0 +1,62 @@
+﻿#if __ANDROID__
+using Android.App;
+using Android.Util;
+using Android.Views;
+using Window = Windows.UI.Xaml.Window;
+
+namespace Uno.UI.Controls
+{
+	public static class NavigationBarHelper
+	{
+		public static bool IsNavigationBarTranslucent
+		{
+			get
+			{
+				var activity = ContextHelper.Current as Activity;
+				var windowFlags = activity.Window.Attributes.Flags;
+
+				return ((int)windowFlags & (int)WindowManagerFlags.TranslucentNavigation) != 0
+					|| ((int)windowFlags & (int)WindowManagerFlags.LayoutNoLimits) != 0;
+			}
+		}
+
+		public static bool IsNavigationBarVisible
+		{
+			get
+			{
+				var activity = ContextHelper.Current as Activity;
+				var systemUiVisibility = activity.Window.DecorView.SystemUiVisibility;
+
+				return (Window.Current.SystemUiVisibility & (int)SystemUiFlags.HideNavigation) == 0
+					|| ((int)systemUiVisibility & (int)SystemUiFlags.HideNavigation) == 0;
+			}
+		}
+
+		public static double LogicalNavigationBarHeight
+		{
+			get
+			{
+				var metrics = new DisplayMetrics();
+				var defaultDisplay = (ContextHelper.Current as Activity)?.WindowManager?.DefaultDisplay;
+
+				if (defaultDisplay == null)
+				{
+					return 0;
+				}
+
+				defaultDisplay.GetMetrics(metrics);
+				var usableHeight = metrics.HeightPixels;
```

Comment:
I would keep the naming consistent with `KeyboardRectProvider.Android.cs`, as to not confuse people.
Previously, we had those code:
https://github.com/nventive/Uno/blob/8bed139b7490f423b4ad93e57df53c26e70d49e3/src/Uno.UI/KeyboardRectProvider.Android.cs#L71-L82

"display" was associated with "GetRect(something...)", while "usable" is linked with `GetWindowVisibleDisplayFrame`.

Reply:
Although, the first 3 lines were removed, the comment is still there. Maybe update the comment to reflect what function they were linked to, and the name they are associated with?

Reply:
Comment changed.
For the first 3 lines, they weren't removed but only moved (only displayRect disappeared). Can you confirm you were talking about these ones ?